Tasks/google-play-release/GooglePlay.ts

```diff
@@ -483,30 +304,26 @@ async function addAllChangelogs(edits: any, apkVersionCodes: any, languageCode:
         return;
```

Comment:
Looks like this method could return `undefined`, the return type should include `| undefined` (or return an empty array here).

Reply:
Good catch.  Thanks!